### DISCUSSION TITLE
Tippy is not defined

### DISCUSSION THREAD

**Discussion by kd75000** [alpinejs/alpine#4142] [7. Bugs (archived)] [answered]
Uncaught ReferenceError: tippy is not defined

Tippy does not work anymore. 

<img width="861" alt="image" src="https://github.com/alpinejs/alpine/assets/62609664/c7254803-df75-4142-ab5f-db454724fd64">

**Comment by joshhanley** [comment, marked as answer]
Unpkg.com CDN is having troubles so it's likely the library isn't being loaded at the moment because of that.
